app/models/manageiq/providers/openshift/container_manager/orchestration_stack.rb

```diff
@@ -0,0 +1,39 @@
+class ManageIQ::Providers::Openshift::ContainerManager::OrchestrationStack < ManageIQ::Providers::ContainerManager::OrchestrationStack
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ContainerManager"
+  belongs_to :container_template, :foreign_key => :orchestration_template_id, :class_name => "ContainerTemplate"
```

Comment:
@moolitayer It will be deleted when the service get retired - a feature will be added later.